reana_client/utils.py

```diff
@@ -64,8 +64,13 @@ def yadage_load(workflow_file, toplevel='.'):
         'schemadir': schemadir,
     }
 
-    return yadageschemas.load(spec=workflow_file, specopts=specopts,
-                              validopts=validopts, validate=True)
+    try:
+        return yadageschemas.load(spec=workflow_file, specopts=specopts,
+                                  validopts=validopts, validate=True)
+
+    except ValidationError as e:
```

Comment:
Could we do this [up in the `validate` command](https://github.com/reanahub/reana-client/blob/758e3e99409a999043e6ca498814b3d541182234/reana_client/cli/workflow.py#L731-L737)? If the fix is casting to string, we can do it up so we fix it in all cases (thinking of CWL validation), what do you think?

Reply:
I didn't want to touch other workflow engines because I don't know what the return value is. That's why I did it only at yadage level. If you think we should move it please let me know. I didn't test other workflow engines as the issue is about yadage.